chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ContactEntity.java

```diff
@@ -69,112 +48,22 @@
     private Date signatureDate;
 
     @Basic(optional = true)
-    @Column(length = NAME_COMPONENT_LENGTH)
+    @Column(length = TITLE_LENGTH)
     private String title;
 
-    /**
-     * Default constructor, mainly for hibernate use.
-     */
-    public ContactEntity() { }
-
-    /**
-     * Constructed with an ID.
-     * @param id the id
-     */
-    public ContactEntity(final Long id) {
-        this.id = id;
-    }
-
-    public Date getCreationDate() {
-        return Util.getNewDate(creationDate);
-    }
-
-    public void setCreationDate(final Date creationDate) {
-        this.creationDate = Util.getNewDate(creationDate);
-    }
-
-    public Boolean getDeleted() {
-        return this.deleted;
-    }
-
-    public void setDeleted(final Boolean deleted) {
-        this.deleted = deleted;
-    }
-
-    public String getEmail() {
-        return this.email;
-    }
-
-    public void setEmail(final String email) {
-        this.email = email;
-    }
-
-    public String getFullName() {
-        return this.fullName;
-    }
-
-    public void setFullName(final String fullName) {
-        this.fullName = fullName;
-    }
-
-    public Long getId() {
-        return this.id;
-    }
-
-    public Date getLastModifiedDate() {
-        return Util.getNewDate(lastModifiedDate);
```

Comment:
Do we still need the `Util.getNewDate` version of the getters/setters?

Reply:
I think we agreed to leave these as removed. There are 907 references to the Util.getNewDate method so I do not intend to remove others at this point.